Tensile/TensileCreateLibrary.py

```diff
@@ -1440,234 +1439,34 @@ def writeMasterFile(
-        dest="ClientConfig",
-        action="store_true",
-        help="Create client config for setting the library and code object files",
-    )
-    argParser.add_argument("--global-parameters", nargs="+", type=splitExtraParameters, default=[])
-    argParser.add_argument(
-        "--ignore-asm-cap-cache",
-        dest="IgnoreAsmCapCache",
-        action="store_true",
-        default=False,
-        help="Ignore asm cap cache and derive the asm caps at runtime",
-    )
-    argParser.add_argument(
-        "--write-master-solution-index",
-        dest="WriteMasterSolutionIndex",
-        action="store_true",
-        default=False,
-        help="Output master solution index in csv format.",
-    )
-    args = argParser.parse_args()
+    lazyLoading = args["LazyLibraryLoading"]
+    separateArchs = args["SeparateArchitectures"]
+    mergeFiles = args["MergeFiles"]
+    embedLibrary = args["EmbedLibrary"]
+    cxxCompiler = args["CxxCompiler"]
+    libraryFormat = args["LibraryFormat"]
+    logicPath = args["LogicPath"]
+    outputPath = args["OutputPath"]
 
-    logicPath = args.LogicPath
-    outputPath = args.OutputPath
-    CxxCompiler = args.CxxCompiler
-    libraryFormat = args.LibraryFormat
     tPrint(3, "OutputPath: %s" % outputPath)
     ensurePath(outputPath)
     outputPath = os.path.abspath(outputPath)
-    arguments = {}
-    arguments["RuntimeLanguage"] = args.RuntimeLanguage
-    arguments["CodeObjectVersion"] = args.CodeObjectVersion
-    arguments["Architecture"] = args.Architecture
-    arguments["SeparateArchitectures"] = args.SeparateArchitectures
-    arguments["LazyLibraryLoading"] = args.LazyLibraryLoading
-    arguments["CxxCompiler"] = args.CxxCompiler
-    if args.CmakeCxxCompiler:
-        os.environ["CMAKE_CXX_COMPILER"] = args.CmakeCxxCompiler
-    arguments["MergeFiles"] = args.MergeFiles
-    arguments["NumMergedFiles"] = args.NumMergedFiles
-    arguments["ShortNames"] = args.ShortNames
-    arguments["LibraryPrintDebug"] = args.LibraryPrintDebug
-    arguments["CodeFromFiles"] = False
-    arguments["EmbedLibrary"] = args.EmbedLibrary
-    arguments["LibraryFormat"] = args.LibraryFormat
-    if args.no_enumerate:
-        arguments["ROCmAgentEnumeratorPath"] = False
-
-    arguments["GenerateManifestAndExit"] = args.GenerateManifestAndExit
-    arguments["VerifyManifest"] = args.VerifyManifest
-
-    arguments["GenerateSourcesAndExit"] = args.GenerateSourcesAndExit
-    if arguments["GenerateSourcesAndExit"]:
-        # Generated sources are preserved and go into output dir
-        arguments["WorkingPath"] = outputPath
-
-    arguments["CpuThreads"] = args.CpuThreads
-    arguments["PrintLevel"] = args.PrintLevel
-    arguments["IgnoreAsmCapCache"] = args.IgnoreAsmCapCache
-    arguments["WriteMasterSolutionIndex"] = args.WriteMasterSolutionIndex
-
-    for key, value in args.global_parameters:
-        arguments[key] = value
-
-    globalParameters["PrintLevel"] = arguments["PrintLevel"]
```

Comment:
Sadly we cannot remove this.

Reply:
Moved this call directly into `assignGlobalParameters`

Reply:
I don't know if this will preserve behavior. What happens to the header prints?